### PR TITLE
V5 style refinements

### PR DESCRIPTION
**Pull request by domyen**
Note: Based on #5426, #5425, #5428 (merge this one first)
Issue: More progress on decomposing stories and components in the UI

## What I did
- Added placeholder stories for Sidebar
- Added Global styles on `body`
- Refactored Sidebar explorer, treeview, and StoriesExplorer into one file
- Removed unused files leftover from Tom's refactor
- Minor tweaks to styles and how we organize Storybook's own stories

### CODE REVIEW

File: lib/ui/src/components/sidebar/SidebarList.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { styled } from '@storybook/theming';
+
+const Heading = styled.span``;
+
+const SidebarList = props => <Heading {...props} />;
+
+export default SidebarList;
```
Comment (tmeasday) [inline lib/ui/src/components/sidebar/SidebarList.js]: What is this component? I think what you call the `SidebarList` is the `StoriesExplorer` which already is built and has these stories

File: lib/ui/src/components/sidebar/SidebarSearch.stories.js [hunk 2]
```diff
@@ -1,18 +1,8 @@
...
-
-const onSetFocussed = action('onSetFocussed');
-export const focussed = () => <PureSidebarSearch focussed onSetFocussed={onSetFocussed} />;
-
-export const filledIn = () => (
-  <PureSidebarSearch focussed onSetFocussed={onSetFocussed} defaultValue="Searchstring" />
-);
+storiesOf('UI|Sidebar/SidebarSearch', module).add('default', () => (
+  <SidebarSearch>TODO SidebarSearch</SidebarSearch>
+));
```
Comment (tmeasday) [inline lib/ui/src/components/sidebar/SidebarSearch.stories.js]: OI! What did you do to my stories!!

**Review by tmeasday — CHANGES_REQUESTED** [review]
Seems good apart from (a) surplus component and (b) crippling of the search?

**Review by tmeasday — APPROVED** [review]

**Comment by ndelangen** [comment]
https://circleci.com/workflow-run/5d2d070c-1e99-4d0d-a191-d5c5c6738d98

**Comment by ndelangen** [comment]
@domyen I think there's a few regressions here, also the fonts look a bit too big, (maybe that's on purpose?)

Do you want to fix them in this branch?

**Comment by domyen** [comment]
I'm working on the styling infra so expect a lot of visual regressions while work is in progress. 

Though I'd prefer to merge into `next` often, it might not make sense given the breadth of visual changes and all the story renaming/editing/removing. How about I keep this branch up to date with `next` and submit the PR when closer to stable? @tmeasday 

@ndelangen Yea I'd prefer to fix them on this branch.

**Comment by shilman** [comment]
@domyen merging this now to get a beta out

**Pull request merged**